CentralManagement/ChocoCCM Command Reference/Remove-CCMDeployment.md

```diff
@@ -0,0 +1,91 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Remove-CCMDeployment
+
+## SYNOPSIS
+Removes a deployment plan
+
+## SYNTAX
+
+```
+Remove-CCMDeployment [-Deployment] <String[]> [-WhatIf] [-Confirm] [<CommonParameters>]
+```

```

Comment:
Do we have an endpoint that functions by ID for this functionality? Deleting purely by name seems like it'd be prone to mistakes.